src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/ConfigurationGeneral.xaml.cs.xlf

```diff
@@ -152,6 +152,11 @@
         <target state="translated">Zakázat rychlou kontrolu aktuálnosti</target>
         <note />
       </trans-unit>
+      <trans-unit id="StringProperty|NETCoreSdkVersion|Description">
+        <source>Represents the version of the dotnet sdk being used (if any).</source>
+        <target state="new">Represents the version of the dotnet sdk being used (if any).</target>
```

Comment:
@tmeschter: We're after string freeze for 15.8, but given that this property is marked as `Visible="False"`, I think it's okay, right?

Reply:
spoke to @tmeschter in person. non-visible properties do not need to be localized